index.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
+
+var loaderUtils = require('loader-utils');
+
+function constructRegex (pragma, exclusive) {
+	var prefix = exclusive ? 'exclude' : 'include';
+	var s = '[\\t ]*\\/\\/>>\\s?' +
+		prefix +
+		'Start\\s?\\(\\s?(["\'])' +
+		pragma +
+		'\\1\\s?,\\s?pragmas\\.' + 
+		pragma + 
```

Comment:
Not really necessary for our current use case, but does pragma need to be escaped here or can it not have special characters to begin with?  Otherwise if the pragma contains a regex character, this will break, right?

Reply:
The pragma needs to be a valid property name to begin with right? (ex `pragmas.debug`) So shouldn't it never have any special characters anyway?

Reply:
Object property names can be anything, as long you you quote it.  For example:

```js
var totallyLegitObject = { 
  "$.\/{}" : "1" 
}
console.log(totallyLegitObject["$.\/{}"]);
```